1-exercises/C-array-every/exercise.js

```diff
@@ -5,7 +5,19 @@
 let students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 let group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-let groupIsOnlyStudents; // complete this statement
+/*function containsStudent (groupWithStudents){
+  for (let name of groupWithStudents){
+    if(group.includes(students)){
+return true;
+    } else{
+      return false;
+    }
+
+  }
+
+}*/
+
+let groupIsOnlyStudents = students.every(name => group.includes(students)? true : false);
```

Comment:
here you don't need the ternary operator `?` because the `.includes()` function already returns a bool